e2e/upgrades.nix

```diff
@@ -67,6 +67,9 @@ in
       union.wait_until_succeeds('[[ $(curl "http://localhost:26657/block" --fail --silent | ${pkgs.lib.meta.getExe pkgs.jq} ".result.block.header.height | tonumber > 1") == "true" ]]')
 
       ${upgradeTo "v1.1.0" 10}
+      ${upgradeTo "v1.2.0" 20}
+      union.wait_until_succeeds('[[ $(curl "http://localhost:26660/block" --fail --silent | ${pkgs.lib.meta.getExe pkgs.jq} ".result.block.header.height | tonumber > 40") == "true" ]]', timeout=120)
```

Comment:
it would be good if we could also verify the contents of the bank module and the delegations here